src/raster/r.sun.daily/r.sun.daily.py

```diff
@@ -248,6 +248,16 @@
 # % description: If not specified, r.sun default will be used.
 # %end
 
+# %option
+# % key: method
+# % type: string
+# % required: no
+# % multiple: no
+# % label: The method to use
```

Comment:
Method for aggregation of daily maps?

Reply:
Changed in f72df2f8330546d4d9af243b4df08af2fd3bd1d2